extensions/service-catalog/client/src/main/resources/resource-operation.vm

```diff
@@ -53,7 +53,7 @@ import java.util.Map;
 import java.util.TreeMap;
 
 
-public class ${model.name}OperationsImpl extends HasMetadataOperation<${model.name}, ${model.name}List, Doneable${model.name}, Resource<${model.name}, Doneable${model.name}>> {
+public class ${model.name}OperationsImpl extends HasMetadataOperation<${model.name}, ${model.name}List${model.name}, Resource<${model.name}${model.name}>> {
```

Comment:
This should probably be 
```suggestion
public class ${model.name}OperationsImpl extends HasMetadataOperation<${model.name}, ${model.name}List${model.name}, Resource<${model.name}>> {
```
instead.

Reply:
Correct! Fixed along with the other issues.